services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -411,13 +428,35 @@ def postgres_db(
 
     yield engine
 
-    assert pg_cli.downgrade.callback
-    pg_cli.downgrade.callback("base")
-    assert pg_cli.clean.callback
-    pg_cli.clean.callback()
-
-    orm.metadata.drop_all(engine)
-    engine.dispose()
+    metadata = MetaData()
+    metadata.reflect(bind=engine)
+    tables = metadata.tables.values()
+
+    # Truncate all the tables
+    with _foreign_key_checks(engine, enabled=False):
```

Comment:
basically you disable foreign key constraints and delete the content of all tables (except the ones you cannot)
- PROS: 
   - probably faster than migration 
- CONS: 
  - exceptional cases need to be added by hand (e.g. everyone group etc)
  - needs an additional test for upgrade/downgrad

Reply:
I would expect one test for upgrade/downgrade.
upgrade is always done at least once per session.
For the rest I agree